### DISCUSSION TITLE
Uncaught TypeError: _block.getKey is not a function at DraftEditorContents.render

### DISCUSSION THREAD

**Discussion by Sassy-Don30** [jpuri/react-draft-wysiwyg#1189] [Q&A]
I was working on a large scale project and tried to use this NPM but it was just giving this error for which I couldn't find a solution. Can someone help me?

**Comment by afilipa** [comment]
Hey @Sassy-Don30 ! 
I was facing the same issue :/
With the hint from https://github.com/jpuri/react-draft-wysiwyg/issues/1005 I realized that immutable dependency was being overwritten to version 4.x due to my project structure.. And checking draft-js package.json it says `"immutable": "~3.7.4"`..

so, reverting the version of immutable to meet the requirements of draft-js fix my issue 

hope this helps you too
